apps/system/style/app_titlebar.css

```diff
@@ -19,7 +14,7 @@
   background-position: bottom right;
 }
 
-#screen:not(.rocketbar-focused):not(.utility-tray):not(.locked) #statusbar.light,
+#screen:not(.rocketbar-focused):not(.utility-tray):not(.locked) .appWindow.active.light > .titlebar .statusbar-shadow,
```

Comment:
Correct me if I'm wrong, but line 17 is a subset of what is covered by line 18 below. Is there a case where line 18 wouldn't match a selector and line 17 would?